src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -112,16 +113,14 @@ trait Formats
     (__ \ "ipAddresses").format[Seq[mesos.NetworkInfo.IPAddress]]
   )(NetworkInfo(_, _, _), unlift(NetworkInfo.unapply))
 
-  implicit lazy val PathIdFormat: Format[PathId] = Format(
-    Reads.of[String](Reads.minLength[String](1)).map(PathId(_)),
-    Writes[PathId] { id => JsString(id.toString) }
-  )
-
-  implicit lazy val PathIdWrites: Writes[PathId] = Writes{ id => JsString(id.toString) }
+  def newPathIdWrites[O <: PathId]: Writes[O] = Writes[O] { id => JsString(id.toString) }
```

Comment:
play-json Format is no longer covariant in it's type parameter. Each type must have a specific format.